helmfile/values/fluentd-user.yaml.gotmpl

```diff
@@ -130,6 +130,11 @@ extraConfigMaps:
       </parse>
     </source>
 
+    <filter **>
```

Comment:
shouldn't we add this filter for fluentd-wc and sc configs?

Reply:
We do not need it in sc, but yeah, i add it in wc also now ofc! :+1: